### PR TITLE
add run script

### PR DESCRIPTION
**Pull request by shunjizhan**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    "echo"
+    "token"
+    "nft"
+  )
+
+  ROOT=$(pwd)
+
+  for e in "${examples[@]}"
+  do
+    echo "--------------- testing truffle ${e} ---------------"
```
Comment (ThunderDeliverer) [inline run.sh:33]: Much like in Waffle tutorials, I'm thinking that we could make the tests just a bit more verbose

```suggestion
    echo "--------------- Running Truffle tests ${e} ---------------"
```

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    -t truffle-tutorial-token \
+    -t truffle-tutorial-nft'
+}
+
+test_all() {
+  examples=(
+    "hello-world"
+    "echo"
+    "token"
+    "nft"
```
Comment (ThunderDeliverer) [inline run.sh:26]: Should these be prefixed with `truffle-tutorial-` or will they execute as expected no matter what?

File: run.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      if ! yarn test-mandala; then
+        ((failed=failed+1))
+      fi
+    fi
+
+    echo ""
+  done
+
+  echo "+++++++++++++++++++++++"
+  echo "truffle test failed: $failed"
```
Comment (ThunderDeliverer) [inline run.sh:51]: Same as above

```suggestion
  echo "Number of failed Truffle tests: $failed"
```

File: run.sh (hunk 2 above)
Comment (shunjizhan) [reply run.sh:26]: they are folder names, so we will loop these and cd them (at line 35)

Comment (shunjizhan) [reply run.sh:26]: so they need to match the actual folder name

**Pull request merged**